### ISSUE TITLE
[bag] Zoom distortion

### ISSUE THREAD

**Issue by dany-pa** [nishanths/loupe-js#10]
## Description

I need to display a small version of the image on the page. It can be absolutely any shape (square, vertical rectangle, horizontal rectangle, circle, etc.). The block that it needs to be placed has dimensions of 200x200. And on hovering, a large picture should be zooming, in which the sides correlate with a small one. And if they are different, then the big picture is distorted.

## What is actually happening?
The desired small image is a square: 
source of small image: [https://i.postimg.cc/NFqgm7XC/small.jpg](https://i.postimg.cc/NFqgm7XC/small.jpg)
![screen1](https://user-images.githubusercontent.com/82453071/115314773-8ab4fb80-a17e-11eb-95cf-46605841c6e7.png)

The large image should be all vertical rectangle. But when zoomed in, it distorts, and each piece of it turns into vertical rectangles.
source of big image: [https://i.postimg.cc/GcXHSW-rz/big.jpg](https://i.postimg.cc/GcXHSW-rz/big.jpg)
![screen](https://user-images.githubusercontent.com/82453071/115314790-930d3680-a17e-11eb-8a25-52c91b1f68da.png)

## Reproduction Link
[https://codepen.io/dany-pa/pen/YzNOmPQ](https://codepen.io/dany-pa/pen/YzNOmPQ)

## How to reproduce
source files:
[style.css](https://github.com/nishanths/loupe-js/blob/master/dist/style.css)
[index.window.js](https://github.com/nishanths/loupe-js/blob/master/dist/index.window.js)
```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <link rel="stylesheet" href="./style.css">
    <script src="index.window.js"></script>
    <style>
        div {
            height: 200px; 
            width: 200px;
        }

        img {
            width: 100%;
            height: 100%;
            object-fit: contain;
            border: 1px solid;
        }
    </style>
</head>
<body>
    <div>
        <img    
            src="https://i.postimg.cc/NFqgm7XC/small.jpg"
            data-zoom="https://i.postimg.cc/GcXHSW-rz/big.jpg"
        >
    </div>
    
    <script>
        const { Loupe, enableLoupe } = window.loupe
        const img = document.querySelector("img")
        const loupe = new Loupe()

        enableLoupe(img, img.dataset.zoom, loupe)
    </script>
</body>
</html>
```

**Comment by dany-pa** [comment]
I know why this happened. 

In file [index.js](https://github.com/nishanths/loupe-js/blob/master/dist/index.js) on lines [68 and 69](https://github.com/nishanths/loupe-js/blob/bf8e6a953a3401a9fb93ddf42d23158b315b9447/dist/index.js#L68) used ```target.getBoundingClientRect()```. 
The original size of small image is 210x200. But target.getBoundingClientRect() returns width: 202 and height 202. Why?
The original size of big image is 793x1362. But the size of the background on the magnifier where the large image is displayed 
 454x454. Because formula on [line 77](https://github.com/nishanths/loupe-js/blob/bf8e6a953a3401a9fb93ddf42d23158b315b9447/dist/index.js#L77) `width * loupe.magnification + 'px ' + height * loupe.magnification + "px"` 

**This is why the image is distorted.**

But so far I cannot figure out how to fix it correctly.

**Comment by nishanths** [comment]
For others arriving here, this would have been fixed in v2.0.0.
